misc/services/acrn_manager/acrnd.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=ACRN manager deamon
 After=systemd-resolved.service
+After=life_mngr.service
```

Comment:
Doesn't that create a hard dependency on a service which isn't installed by default?

What happens if you *just* do a `make install` on a target and reboot, does the `acrnd` service start successfully although `life_mngr` is not installed?

Reply:
also the correct spelling is "daemon"

Reply:
> Doesn't that create a hard dependency on a service which isn't installed by default?
> 
> What happens if you _just_ do a `make install` on a target and reboot, does the `acrnd` service start successfully although `life_mngr` is not installed?
This patch wants to  load acrnd only  if life_mngr.service has been loaded.

Reply:
> also the correct spelling is "daemon"

sure

Reply:
> > Doesn't that create a hard dependency on a service which isn't installed by default?
> > What happens if you _just_ do a `make install` on a target and reboot, does the `acrnd` service start successfully although `life_mngr` is not installed?

> This patch wants to  load acrnd only  if life_mngr.service has been loaded.

OK, so the `acrnd` daemon will not start for as long as the `life_mngr` daemon isn't running, is that correct?

Reply:
yes

Reply:
This is a problem that we need to fix.

Reply:
> This is a problem that we need to fix.

Hi gvancuts, I'm sorry to mislead you. I ask xiangyang, he said acrnd daemon would start even the life_mngr daemon isn't running.

Reply:
> Hi gvancuts, I'm sorry to mislead you. I ask xiangyang, he said acrnd daemon would start even the life_mngr daemon isn't running.

OK, thanks for the clarification